glean-core/rlb/src/lib.rs

```diff
@@ -441,6 +442,64 @@ pub(crate) fn submit_ping_by_name_sync(ping: &str, reason: Option<&str>) {
     }
 }
 
+/// Indicate that an experiment is running.  Glean will then add an
+/// experiment annotation to the environment which is sent with pings. This
+/// infomration is not persisted between runs.
+///
+/// See `glean_core::Glean.set_experiment_active`
+pub fn set_experiment_active(
+    experiment_id: String,
+    branch: String,
+    extra: Option<HashMap<String, String>>,
+) {
+    dispatcher::launch(move || {
+        with_glean(|glean| {
+            glean.set_experiment_active(
+                experiment_id.to_owned(),
+                branch.to_owned(),
+                extra.to_owned(),
+            )
+        });
+    })
+}
+
+/// Indicate that an experiment is no longer running.
+///
+/// See `glean_core::Glean.set_experiment_inactive`
+pub fn set_experiment_inactive(experiment_id: String) {
+    dispatcher::launch(move || {
+        with_glean(|glean| glean.set_experiment_inactive(experiment_id.to_owned()))
+    })
+}
+
+/// TEST ONLY FUNCTION.
+/// Checks if an experiment is currently active.
+#[cfg(test)]
+#[allow(dead_code)]
```

Comment:
Note that this is part of the *public* testing API that RLB consumer should be able to use, so it can't be restricted with `#[cfg(test)]` (see [booleans](https://github.com/mozilla/glean/blob/197ddb7f1ea309f7d026ae814728101a3e151336/glean-core/rlb/src/private/boolean.rs#L54)).

The same reasoning applies to the other things :)